app/services/applied_add_ons/create_service.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module AppliedAddOns
+  class CreateService < BaseService
+    def create(**args)
+      @customer = Customer.find_by(
```

Comment:
I'm more into use `current_customer`, I'm not a big fan of instance classes into public code, wdyt?

```ruby
def current_customer
  @current_customer ||= Customer.find_by...
end
```